src/sentry/south_migrations/0470_org_saved_search.py

```diff
@@ -0,0 +1,1374 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+from sentry.utils.db import is_postgres
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = True
```

Comment:
@mattrobenolt Not sure if you want to take a look at this migration. Otherwise I can just generate the ddl and hand off to #ops for tomorrow. Nothing too exciting here, creating the FK to organization worked fine earlier.